package.json

```diff
@@ -14,26 +14,27 @@
     "Firefox ESR"
   ],
   "stack": {
-    "name": "V081TextractDemo",
-    "region": "us-west-2"
+    "name": "DUS",
+    "region": "%%REGION%%"
   },
+  "email": "%%USER_EMAIL%%",
   "scripts": {
     "compile-ts-stack": "tsc deployment/custom-deployment/lib/cdk-textract-stack.ts --target es2018 --module commonjs --allowjs true",
     "compile-ts-client-stack": "tsc deployment/custom-deployment/lib/cdk-textract-client-stack.ts --target es2018 --module commonjs --allowjs true",
-    "deploy:stack": "yarn compile-ts-stack && AWS_REGION=$npm_package_stack_region cdk deploy -v true",
-    "deploy:client": "yarn compile-ts-client-stack && yarn export && yarn update-client-stack-variables && AWS_REGION=$npm_package_stack_region cdk deploy -v true -a \"node bin/deploy-client-stack.js\" && yarn update-pdf-lambda-code",
+    "deploy:stack": "yarn compile-ts-stack && AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk deploy --require-approval never -v true",
```

Comment:
NIT : I am wondering if we should we call this deploy:backend or deploy:server since deploy:stack could make customers think that it deploys the entire stack (demo and client)

Reply:
I agree, will update.